### PR TITLE
Fix language in docs/sources/panels/queries.md

### PR DESCRIPTION
**Pull request by Mohit-Nain**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/panels/queries.md [hunk 1]
```diff
@@ -82,25 +82,25 @@ Panel data source query options:
 
 - **Max data points -** If the data source supports it, sets the maximum numbers of data points for each series returned. If the query returns more data points than the max data points setting, then the data source consolidates them (reduces the number of points returned by aggregating them together by average or max or other function).
   
-  There are two main reasons for limiting the number of points, performance and smoothing the line. The default value is the width (or number of pixels) of the graph as there is no point in having more data points than the graph panel can display.
+  There are two main reasons for limiting the number of points, performance and smoothing the line. The default value is the width (or number of pixels) of the graph as there is no point in having more data points than, what the graph panel can display.
```
Comment (aknuds1) [inline docs/sources/panels/queries.md:85]: Please undo this change.

File: docs/sources/panels/queries.md [hunk 2]
```diff
@@ -82,25 +82,25 @@ Panel data source query options:
...
 
   With streaming data, the max data points value is used for the rolling buffer. (Streaming is a continuous flow of data and buffering is a way of dividing the stream into chunks). Loki streams data in the live tailing mode.
 
-- **Min interval -** Sets a minimum limit for the automatically calculated interval, typically the minimum scrape interval. If a data point is saved every 15 seconds, then there's no point in having an interval lower than that. Another use case is set it to a higher minimum than the scrape interval to get more coarse-grained, well-functioning queries.
+- **Min interval -** Sets a minimum limit for the automatically calculated interval, typically the minimum scrape interval. If a data point is saved every 15 seconds, then there's no point in having an interval lower than that. Another use case is, set it to a higher minimum than the scrape interval to get more coarse-grained, well-functioning queries.
```
Comment (aknuds1) [inline docs/sources/panels/queries.md:89]:
```suggestion
- **Min interval -** Sets a minimum limit for the automatically calculated interval, typically the minimum scrape interval. If a data point is saved every 15 seconds, then there's no point in having an interval lower than that. Another use case is to set it to a higher minimum than the scrape interval to get more coarse-grained, well-functioning queries.
```

File: docs/sources/panels/queries.md [hunk 3]
```diff
@@ -82,25 +82,25 @@ Panel data source query options:
...
 
   For more information, refer to [Global variables]({{< relref "../variables/global-variables.md" >}}).
 
 - **Relative time -** You can override the relative time range for individual panels, causing them to be different than what is selected in the dashboard time picker in the top right corner of the dashboard. This allows you to show metrics from different time periods or days on the same dashboard.
   
 - **Time shift -** The time shift function is another way to override the time range for individual panels. It only works with relative time ranges and allows you to adjust the time range. 
   
-  For example, you could shift the time range for the panel to be two hours earlier than the dashboard time picker. For more information, refer to [Time range controls]({{< relref "../dashboards/time-range-controls.md" >}}).
+  For example, you can shift the time range for the panel to be two hours earlier than the dashboard time picker. For more information, refer to [Time range controls]({{< relref "../dashboards/time-range-controls.md" >}}).
```
Comment (aknuds1) [inline docs/sources/panels/queries.md:103]: Please undo this change.

**Review by aknuds1 — CHANGES_REQUESTED** [review]

File: docs/sources/panels/queries.md (hunk 3 above)
Comment (Mohit-Nain) [reply docs/sources/panels/queries.md:103]: Can makes a better read as we are referring to something in present, could is generally used while referring to past .

File: docs/sources/panels/queries.md (hunk 1 above)
Comment (Mohit-Nain) [reply docs/sources/panels/queries.md:85]: i think the change makes the document better in terms of readability, if we do not make the change, the document might look like eating on words

Comment (aknuds1) [reply docs/sources/panels/queries.md:85]: Sorry, but I don't agree that this change improves the readability.

File: docs/sources/panels/queries.md (hunk 3 above)
Comment (aknuds1) [reply docs/sources/panels/queries.md:103]: Sorry, but I think the original language is fine.

**Review by aknuds1 — CHANGES_REQUESTED** [review]

**Review by aknuds1 — APPROVED** [review]

**Comment by aknuds1** [comment]
Thank you for contributing to Grafana!
